pkg/test/fake.go

```diff
@@ -56,26 +56,41 @@ func NewMockClient() *MockClient {
 
 // Get calls MockClient's MockGet function.
 func (c *MockClient) Get(ctx context.Context, key client.ObjectKey, obj runtime.Object) error {
+	if c.MockGet == nil {
```

Comment:
That's a fair point. I added this clause simply to avoid writing that stub function out over and over again while writing tests, but I agree it's better for a nil pointer to be returned and blow up the tests if a function is called unexpectedly.

One compromise would be to define the stub function as a `DefaultMockGet` variable (along with `DefaultMockUpdate`, etc). This way the functions could be `nil` by default but also have a tiny bit less boilerplate.

Reply:
> A few comments and one change request.

@ichekrygin Just to clarify, this was the thing you felt strongly must be changed right?

Reply:
Yes, please, I would prefer to "undo" this change since it may be affecting existing tests.